src/libraries/Common/tests/System/Net/Http/Http3LoopbackConnection.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace System.Net.Test.Common
 {
-    public sealed class Http3LoopbackConnection : GenericLoopbackConnection
+    internal sealed class Http3LoopbackConnection : GenericLoopbackConnection
```

Comment:
Why make this internal? It's a test project.

Reply:
It referenced the now internal Quic types in 'public' API.